packages/renderer/src/lib/events/EventUI.ts

```diff
@@ -0,0 +1,32 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { V1ObjectMeta } from '@kubernetes/client-node';
+
+export interface EventUI {
+  count?: number;
+  apiVersion?: string;
+  kind?: string;
+  metadata?: V1ObjectMeta;
```

Comment:
Some field feel unnecessary to me, why do  we need metadata property ? why don't we have `namespace` and `name` just like DeploymentUI ?

Reply:
right, I first added them to make the structure a superset of KubernetesObject, but it is not necessary. I'm removing them